source/workshop_1_mcmc_en_brms.Rmd

```diff
@@ -203,15 +203,167 @@ Er zijn twee mogelijke oplossingen:
   - de conjugate priors beperken de vrijheid om je voorkennis voor te stellen. (bijvoorbeeld zero-inflation, )
 - *Sampling*: we krijgen informatie over de verdeling door eruit te samplen in plaats van het exact te berekenen.
 
-#--------------------------------INSERT MCMC HERE------------------------------#
-laat ze zelf een chain runnen (buiten brms?)
-Chains, iterations, burn-in -> https://nicercode.github.io/guides/mcmc/
-Rhat
-Parallel running
-Stan
-Verschillende regels metropolis-hastings algoritme
-MCMC: vermeld verdunning (thinning), chains, burn-in, iterations
-#--------------------------------END MCMC HERE---------------------------------#
+## Parameterschatting met MCMC
+
+### Wat is MCMC?
+
+MCMC staat voor 'Monte Carlo' 'Markov Chain'. We gaan niet dieper in op deze begrippen. Gaandeweg wordt wel duidelijk wat hiermee wordt bedoeld.
+
+De posterior distributie kan je vergelijken met een berg. De top van de berg ligt waar de parameterwaarde het meest waarschijnlijk is (maximum likelihood). De ligging en de vorm van de bergtop is echter onbekend. Het doel van een MCMC algortime is op een efficiënte wijze de bergtop en de omgeving te verkennen. 
+
+Een MCMC genereert een reeks van parameterwaarden aan de hand van een aantal regels die de sampler toelaat de hele range van de posterior distributie te exploreren. De regels zorgen ervoor dat de sampler de hele tijd in de buurt van de top van de berg blijft.
+
+Een MCMC is een random wandeling in de parameterruimte in de omgeving waar de posterior distributie het hoogst is (de top van de berg). De grootte en de richting van de stap varieert.
+
+Bij een stap bergop gaat de MCMC altijd door. Is de stap bergaf, dan gaat de MCMC niet altijd door. De kans om bergaf te gaan hangt af van hoe diep de stap naar beneden is. Kleine stapjes naar beneden worden vaak genomen. Bij diepe stappen naar benenden blijft de MCMC meestal staan en kiest de MCMC vervolgens een nieuwe stap.
+
+De absolute hoogte van de berg (posterior likelihood) doet er op zich niet toe. Enkel de relatieve verschillen.
+
+![Metropolis regels](../media/MCMC4b.png)
+
+Een eenvoudig MCMC algoritme is het Metropolis algoritme. Het wordt beschreven door deze formele regels:
+
+1. Kies een startwaarde voor de parameter
+2. Bereken voor die parameter de posterior likelihood (= prior + likelihood)
+3. Kies lukraak een nieuwe parameterwaarde, maar 'in de buurt' van de vorige
+4. Bereken de nieuwe posterior voor die nieuwe parameterwaarde
+5. Is de likelihood voor de nieuwe posterior > huidige posterior => nieuwe waarde behouden
+6. Is de likelihood voor de nieuwe posterior < huidige posterior =>
+   a) Bereken de ratio (nieuwe posterior / vorige posterior.
+   b) Aanvaard de nieuwe waarde met een kans gelijk aan de ratio. (dus hoe lager de nieuwe likelihood hoe kleiner de kans op aanvaarden. 
```

Comment:
b) Aanvaard de nieuwe waarde met een kans gelijk aan de ratio. (dus hoe lager de nieuwe likelihood hoe kleiner de kans op aanvaarden. 

--> eerder "hoe lager de nieuwe posterior"? (onrechtstreeks komt dit waarschijnlijk wel op hetzelfde neer?

Reply:
Ja, nieuwe posterior is inderdaad correcter. De posterior is immers som van likelihood prior + likelihood data